packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -887,10 +903,10 @@ export function diffHydratedProperties(
   assertValidProps(tag, rawProps, getStack);
 
   if (__DEV__) {
-    var extraAttributeNames: Set<string> = new Set();
```

Comment:
Why was this extracted to top level? Looks like a local variable to me.

Reply:
hm, you right @gaearon. Updated :octocat: